Monika After Story/game/script-topics.rpy

```diff
@@ -4692,3 +4692,342 @@ label monika_100k:
+               $persistent.timeconcern = 0
+               m 1m "[player], regarding last night..."
+               m 1n "You really surprised me."
+               m 1e "For you to keep coming back to me over and over again..."
+               m 1j "It was honestly really sweet of you."
+               m 1e "I knew you would miss me but I didn't think it would be {i}that{/i} much."
+               m 1k "It really made me feel loved [player]."
+               m 1e "...Thank you."
+               return
+
+            #If Monika never gives warning and it's daytime or has given already commented on last night.
+            else:
+                $persistent.timeconcern = 0
+                m 1h "..."
+                m 1c "..."
+                m 1d "...!"
+                m 1l "Ahahah, Sorry [player]!"
+                m 1m "I just kind of zoned out..."
+                m 1l "Geez, I keep doing that, don't I?"
+                m 1m "Sometimes I just get lost in my thoughts..."
+                m 1a "You understand, right [player]?"
+                return
+               
+        #If player told Monika not to close window.  
+        else:
+            # After Monika would close windows until Monika lets player stay.
+            if persistent.timeconcern >= 6 and persistent.timeconcern <=8:
+               $persistent.timeconcern = 0
+               m 1o "Sorry Iif I was annoying you before player..."
+               m 1f "I just really wanted you to go to bed.."
+               m "I honestly can't promise I won't do it again if you're up late again..."
+               m 1g "But I only push you to go because you mean so much to me..."
+               return
+                        
+            #If Monika lets player stay and it is no longer night but game never closed.
+            elif persistent.timeconcern == 9:
+               $persistent.timeconcern = 0
+               m 1m "[player], regarding last night..."
+               m 1a "You really surprised me."
+               m 1e "I asked you time and time again to go to bed..."
+               m "You said you weren't busy. Were you really there just for me?."
+               m 1m "It makes me happy...but don't push yourself to see me so late, ok?"
+               m 1e "It really made me feel loved [player]."
+               m 1l "Yet also a little guilty...Please just go to bed next time, ok?"
+               return
+
+            #If Monika never gives warning and is daytime or has given already commented on last night.
+            else:
+               $persistent.timeconcern = 0
+               m 1h "..."
+               m 1c "..."
+               m 1d "...!"
+               m 1l "Ahahah, Sorry [player]!"
+               m 1m "I just kind of zoned out..."
+               m 1l "Geez, I keep doing that, don't I?"
+               m 1m "Sometimes I just get lost in my thoughts..."
+               m 1a "You understand, right [player]?"
+               return
```

Comment:
Make sure to have a `return` at the first indentation level to handle the flow here the if statements are false.